src/DssSpell.sol

```diff
@@ -65,82 +57,86 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
+        DssExecLib.addReaderToWhitelist(DssExecLib.getChangelogAddress("PIP_UNI"),    OASIS_APP_OSM_READER);
+        DssExecLib.addReaderToWhitelist(DssExecLib.getChangelogAddress("PIP_LINK"),   OASIS_APP_OSM_READER);
+        DssExecLib.addReaderToWhitelist(DssExecLib.getChangelogAddress("PIP_MANA"),   OASIS_APP_OSM_READER);
+
+        // --------------------------------- Replace CRVV1ETHSTETH-A PIP -----------------------------------
+        bytes32 _ilk  = "CRVV1ETHSTETH-A";
+
+        address PIP_CRVV1ETHSTETH_OLD = DssExecLib.getChangelogAddress("PIP_CRVV1ETHSTETH");
+        address MCD_CLIP_CRVV1ETHSTETH_A = DssExecLib.getChangelogAddress("MCD_CLIP_CRVV1ETHSTETH_A");
+
+        // OSM Sanity Checks
+        require(CurveLPOracleLike(PIP_CRVV1ETHSTETH).pool() == CurveLPOracleLike(PIP_CRVV1ETHSTETH_OLD).pool(), "DssSpell/pip-wrong-pool");
+        require(CurveLPOracleLike(PIP_CRVV1ETHSTETH).src() == CurveLPOracleLike(PIP_CRVV1ETHSTETH_OLD).src(), "DssSpell/pip-wrong-src");
+        require(CurveLPOracleLike(PIP_CRVV1ETHSTETH).wat() == CurveLPOracleLike(PIP_CRVV1ETHSTETH_OLD).wat(), "DssSpell/pip-wrong-wat");
+        require(CurveLPOracleLike(PIP_CRVV1ETHSTETH).ncoins() == CurveLPOracleLike(PIP_CRVV1ETHSTETH_OLD).ncoins(), "DssSpell/pip-wrong-ncoins");
+        require(CurveLPOracleLike(PIP_CRVV1ETHSTETH).orbs(0) == CurveLPOracleLike(PIP_CRVV1ETHSTETH_OLD).orbs(0), "DssSpell/pip-wrong-orbs0");
+        require(CurveLPOracleLike(PIP_CRVV1ETHSTETH).orbs(1) == CurveLPOracleLike(PIP_CRVV1ETHSTETH_OLD).orbs(1), "DssSpell/pip-wrong-orbs1");
+
+        address OSM_MOM = DssExecLib.osmMom();
+        address MCD_SPOT = DssExecLib.spotter();
+        address CLIPPER_MOM = DssExecLib.clipperMom();
+        address MCD_END = DssExecLib.end();
+
+        // Revoke OsmMom to access the Old OSM
+        DssExecLib.deauthorize(PIP_CRVV1ETHSTETH_OLD, OSM_MOM);
 
+        // Remove Old CRVV1ETHSTETH-A PIP Whitelistings
+        DssExecLib.removeReaderFromWhitelistCall(CurveLPOracleLike(PIP_CRVV1ETHSTETH_OLD).orbs(0), PIP_CRVV1ETHSTETH_OLD);
+        DssExecLib.removeReaderFromWhitelistCall(CurveLPOracleLike(PIP_CRVV1ETHSTETH_OLD).orbs(1), PIP_CRVV1ETHSTETH_OLD);
+
+        DssExecLib.removeReaderFromWhitelist(PIP_CRVV1ETHSTETH_OLD, MCD_SPOT);
+        DssExecLib.removeReaderFromWhitelist(PIP_CRVV1ETHSTETH_OLD, MCD_CLIP_CRVV1ETHSTETH_A);
+        DssExecLib.removeReaderFromWhitelist(PIP_CRVV1ETHSTETH_OLD, CLIPPER_MOM);
+        DssExecLib.removeReaderFromWhitelist(PIP_CRVV1ETHSTETH_OLD, MCD_END);
+
+        // ---- Replace CRVV1ETHSTETH-A PIP ----
+
+        // Set the token PIP in the Spotter
+        DssExecLib.setContract(MCD_SPOT, _ilk, "pip", PIP_CRVV1ETHSTETH);
+
+        // Allow OsmMom to access the New OSM
+        DssExecLib.authorize(PIP_CRVV1ETHSTETH, OSM_MOM);
+
+        // Add New CRVV1ETHSTETH-A PIP Whitelistings
+        DssExecLib.addReaderToWhitelistCall(CurveLPOracleLike(PIP_CRVV1ETHSTETH).orbs(0), PIP_CRVV1ETHSTETH);
```

Comment:
We can use the standard `addReaderToWhitelist()` here since we know these orbs exist. The execlib falls back to call because it's cheaper to just attempt a kiss than to process the logic to determine whether the pip is an OSM or DsValue.

Reply:
done in ea876c7